docker/docker-compose.yml

```diff
@@ -46,6 +47,13 @@ services:
     ports:
       - '5435:5432'
 
+  cockroachdb:
+    image: prismagraphql/build:cockroach-custom
+    restart: always
+    command: start-single-node --insecure
+    ports:
+      - '26257:26257'
```

Comment:
Should we also attach a volume to persist data similar to other databases here? LGTM either way though.

Reply:
What do you persist the data for?

Reply:
(Might just be a think we have been cargo culting with no actual use...)

Reply:
Yeah, on the second thought I would rather prefer we don't persist data in the other services too instead.